src/System.ComponentModel.Annotations/tests/MaxLengthAttributeTests.cs

```diff
@@ -111,20 +112,13 @@ public static void GetValidationResult_ValueNotStringOrICollection_ThrowsInvalid
         [Fact]
         public static void GetValidationResult_ValueGenericICollection_ThrowsInvalidCastException()
```

Comment:
Could you change the  name of the test method (plus the one below) and add a couple of tests - one for `ICollection` and one for `ICollection<T>`?

Reply:
Will do.

Reply:
Done.